ext/symengine/symengine.c

```diff
@@ -44,6 +45,9 @@ void Init_symengine() {
     rb_define_method(c_basic, "subs", cbasic_subs, -1);
     rb_define_method(c_basic, "coerce", cbasic_coerce, 1);
 
+    //Sympify as a Module Level Function
+    rb_define_module_function(m_symengine, "sympify", cutils_sympify, 1);
```

Comment:
`convert` sounds good. Since `coerce` is used in ruby for a similar thing, it's better not to use that.
Do you mean `SymEngine(expression)` or `SymEngine::SymEngine(expression)`? If it's the former and can be done in Ruby, I'm +1 for that.